platforms/Windows/Directory.Build.props

```diff
@@ -24,34 +29,58 @@
   <PropertyGroup>
     <RootBuildFolder>$([System.IO.Path]::GetFullPath($(MSBuildThisFileDirectory)build\))</RootBuildFolder>
     <BaseOutputPath>$(RootBuildFolder)</BaseOutputPath>
+    <BaseOutputPath Condition="!HasTrailingSlash('$(BaseOutputPath)')">$(BaseOutputPath)\</BaseOutputPath>
     <BaseIntermediateOutputPath>$(BaseOutputPath)obj\$(MSBuildProjectName)\</BaseIntermediateOutputPath>
     <OutputPath>$(BaseOutputPath)$(Configuration)\$(ProductArchitecture)\</OutputPath>
 
-    <DefaultCompressionLevel Condition="'$(DefaultCompressionLevel)' == ''">high</DefaultCompressionLevel>
+    <BundleFlavor Condition="'$(BundleFlavor)' == ''">web</BundleFlavor>
```

Comment:
Might be nice to make this `online` (to be an obvious inversion of `offline`).

Reply:
Done.